README.md

```diff
@@ -1,13 +1,10 @@
-[![Build Status](https://travis-ci.org/Mathmagicians/kumori.svg?branch=master)](https://travis-ci.org/Mathmagicians/kumori) 
+[![Build Status](https://travis-ci.org/Mathmagicians/kumori.svg?branch=master)](https://travis-ci.org/Mathmagicians/kumori)
+[![Sonar Status](https://sonarcloud.io/api/project_badges/measure?project=Mathmagicians_kumori&metric=alert_status)](https://sonarcloud.io/dashboard?id=Mathmagicians_kumori)
 
 <img src="https://p14.zdusercontent.com/attachment/1015988/ILt64wv09EiSYlczYn9qT6sbO?token=eyJhbGciOiJkaXIiLCJlbmMiOiJBMTI4Q0JDLUhTMjU2In0..p5isVLQUoFjBFQxrbKuZ7A.pd5GzuefqEFeVonPbUT2RXV9zfcjOMeceoWh-DvzrUaKpmmoJ3bfFFsRGo3_hkwsz0WUlLCfibURlXnD3TUIuUarpqcEIScqbRDRImkgp8yp8pMrVsldZVcx3-pdyY0JsssrpGMxVBFEF-0sRpUU5fpcLLaR1SC9RjPsN7fRroQ45UOpYbfntVwaAIeEiKcVbqW4DSIyGJ2yqsYB4JCXKbo-kmLv9Y2huo87j4qbZnnvAUV2y99_QpPcl5KRDvsy6L4qXBasZU4bHa7YnE83CK5PphMORCfcAT7pAk9tgrU.58aljhq1tzvjb-BavJPb8w" width="100">
 
 # Kumori
 
 Healthcheck of your architecture.
 
-This projects uses [Browserstack](https://www.browserstack.com) for automated testing. 
-
-
-
-
+This projects uses [Browserstack](https://www.browserstack.com) for automated testing.
```

Comment:
I am missing a description of the purpose of the tools (what problem ...) for sonarqube and browserstack
what do they do for us - any usage policies?

Reply:
This reference is added as a request from browserstack in exchange for supporting the project with a free usage of there service. I have yet to set the test up.